osf/migrations/0107_add_dependent_styles.py

```diff
@@ -148,5 +148,4 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(update_styles, revert),
```

Comment:
Ditto, is there a reason to keep `def update_styles(state, schema): and `def get_style_files(path):`?

Reply:
Ditto, I realized we don't remove them for other PRs. Thus I took my suggestion back. Consider resolved.